test/RibbonThetaVault.js

```diff
@@ -2665,13 +2691,115 @@ function behavesLikeRibbonOptionsVault(params) {
           depositAmount
         );
 
-        // Only 1 ether - MINIMUM_SUPPLY works
+        // only 1 ether - MINIMUM_SUPPLY works
         await expect(
           this.vault.withdraw(depositAmount.sub(BigNumber.from("1")))
         ).to.be.revertedWith(/Insufficient share supply/);
       });
     });
 
+    describe("#withdrawToV1Vault", () => {
+      time.revertToSnapshotAfterEach();
+
+      it("should fail if vaults aren't registered", async function () {
+        const depositAmount = BigNumber.from("100000000000");
+        await depositIntoVault(
+          params.collateralAsset,
+          this.vault,
+          depositAmount
+        );
+
+        await expect(
+          this.vault.withdrawToV1Vault(
+            this.vault.maxWithdrawableShares(),
+            this.spareVault.address
+          )
+        ).to.be.revertedWith("!feeless");
+      });
+
+      it("should transfer without fee", async function () {
+        const depositAmount = BigNumber.from("100000000000");
+        await depositIntoVault(
+          params.collateralAsset,
+          this.vault,
+          depositAmount
+        );
+
+        await this.registry.registerFreeWithdrawal(
+          this.vault.address,
+          this.spareVault.address
+        );
+
+        await this.vault.withdrawToV1Vault(
+          BigNumber.from("10000000000"),
+          this.spareVault.address
+        );
+        // No fee deposited back into 1st vault
+        assert.equal(
+          (await this.assetContract.balanceOf(this.vault.address)).toString(),
+          BigNumber.from("90000000000").toString()
+        );
+        assert.equal(
+          (await this.vault.balanceOf(user)).toString(),
+          BigNumber.from("90000000000").toString()
+        );
+        assert.equal(
+          (
+            await this.assetContract.balanceOf(this.spareVault.address)
+          ).toString(),
+          BigNumber.from("10000000000").toString()
+        );
+        assert.equal(
+          await this.spareVault.balanceOf(user),
+          BigNumber.from("10000000000").toString()
+        );
```

Comment:
assert that `this.spareVault.balanceOf(this.vault.address) == 0`

Reply:
Good idea